plugin-name/assets/src/plugin-settings.js

```diff
@@ -0,0 +1,9 @@
+import './styles/settings.scss';
```

Comment:
Nope i guess :) This is required to webpack in order to compile scss into css. 
In this case will be generated the "plugin-settings.css" because is included in "plugin-settings.js", and that's the reason why I've included the xyz.scss file for each js file (this is valid also for following reviews).